README.md

```diff
@@ -1,18 +1,18 @@
-# hackpsuS2018
-Repository for the main website for HackPSU Spring 2018
+##HackPSU Fall 2019
```

Comment:
This actually needs the space in front for the formatter. You can double check it if you click on the "Display rich diff" button in the top right.

Reply:
Probably also only one hashtag

Reply:
Probably also only needs one hashtag

Reply:
Fixed

Reply:
Done

Reply:
I already said that lmao